src/index.js

```diff
@@ -1,9 +1,11 @@
-const interfaces = require('./interfaces.js');
+'use strict';
 
-const ENS = require('ethereum-ens');
+var interfaces = require('./interfaces.js');
```

Comment:
Based on the changes in this file, it almost looks like it's been processed by babel to ES5?